src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -5157,6 +5255,283 @@ namespace AirflowNetworkBalanceManager {
 		return CalcWindPressure;
 	}
 
+	Real64
+	AirDensity(
```

Comment:
Since this and the other correlations are new functions, let's use a lower case first letter (`airDensity`, `airKinematicViscosity`, etc.). Also, using this function is going to clash with other parts of the code, where `PsyRhoAirFnPbTdbW` is used to get the air density.